### PR TITLE
Create commonErrors.md

### PR DESCRIPTION
**Pull request by MayBeKurito**
+ Common functions errors
+ Exclusive function errors
+ Common discord API errors
Co-authored:
@Rainb0wKey

### CODE REVIEW

**Comment by Kitosight** [comment]
This needs a lot of editing.

**Comment by MayBeKurito** [comment]
> This needs a lot of editing. 

I read you...

File: src/guides/other/commonErrors.md [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+
+The argument flags mark if arguments within a function may be used (or hence, not used at all).
+
+> You can check the argument flags list [here](https://nilpointer-software.github.io/bdfd-wiki/guides/other/arguments.html#argument-flags).
+
+---
+
+# Expectations
+
+## Aclaration
```
Comment (MineBartekSA) [inline src/guides/other/commonErrors.md:48]: What is "Aclaration"?

File: src/guides/other/commonErrors.md [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+# Error Messages Handling 
+
+## Error message syntax
+
+### Error function
+
+The function that is causing the error.
+
+**Function location:**
+![Location of function](https://media.discordapp.net/attachments/1021982737424449586/1255928553921576992/syntax_msg_error-function.png?ex=667eea1d&is=667d989d&hm=295240ddc00f1c0a97f2a87940bab47d691b958b21c6b23ede29f61da45542d4&)
```
Comment (MineBartekSA) [inline src/guides/other/commonErrors.md]: Please update image

File: src/guides/other/commonErrors.md [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+
+### Row and Column
+
+- `Row`: 
+The row where the error is located.
+- `Column`:
+The column where the error is located.
+
+**Row and column location:**
+![Location of row and column](https://media.discordapp.net/attachments/1021982737424449586/1255955254789537852/syntax_msg_error-rowcolumn.png?ex=667f02fb&is=667db17b&hm=4ca5a29b2576e824e65db688973e662549f5e721b157b88f8cee4920b529bb97&)
```
Comment (MineBartekSA) [inline src/guides/other/commonErrors.md]: Please update image

File: src/guides/other/commonErrors.md [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+</details>
+
+
+
+
+
+
+
+
+
```
Comment (MineBartekSA) [inline src/guides/other/commonErrors.md]: Please only leave one empty line at the end of files

**Review by MineBartekSA — CHANGES_REQUESTED** [review]

File: src/guides/other/commonErrors.md (hunk 1 above)
Comment (MayBeKurito) [reply src/guides/other/commonErrors.md:48]: Is a way of saying "Making it clear" or "Clarifying".

Comment (MineBartekSA) [reply src/guides/other/commonErrors.md:48]: Alright, I will take your word for it since I have not been able to find this word in any dictionary.

**Review by MineBartekSA — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**